### ISSUE TITLE
Edit wiki: stories include angular material

### ISSUE THREAD

**Issue by damondouglas** [angular/angular-cli#7236]
### Bug Report or Feature Request (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [ ] feature request
- [x] wiki edit request
```

### Versions.

```bash
ng --version
@angular/cli: 1.2.6
node: 8.0.0
os: darwin x64
```

### Desired wiki edits.

Please see [commit](https://github.com/damondouglas/angular-cli-wiki/commit/c8672976cca3c1429182888351a9b83aea29ca6b) on my personal repo for details.

### Mention any other details that might be useful.

Following instructions in [stories include angular material](https://github.com/angular/angular-cli/wiki/stories-include-angular-material) results in #[5320](https://github.com/angular/material2/issues/5320) resolved by [comment by w3chtml](https://github.com/angular/material2/issues/5320#issuecomment-313656177).

**Comment by filipesilva** [comment]
Heya, would you be available to add a PR to fix this?

The file you should edit is https://github.com/angular/angular-cli/blob/master/docs/documentation/stories/include-angular-material.md. We then publish it to the wiki on releases.

A good commit message would be `docs: fix material install`. 

I ask you don't include the extra yarn bit though. I know a lot of users use yarn but I don't expect it to be a confusion point (yarn users know the correspondence), and we generally do not add both instructions throughout the docs.

**Comment by filipesilva** [comment]
Closed via #7254
